app/uploaders/base_uploader.rb

```diff
@@ -26,6 +26,9 @@ def size_range
 
   # strip EXIF (and GPS) data
   def strip_exif
+    # svg's do not contain exif or gps data
+    return if file.content_type.include?("svg")
```

Comment:
SVG's do not contain exif or gps data so I returned early. 

We also know that SVG files can contain JavaScript, and are thus very vulnerable for injection attacks.
As per a discussion with @nickytonline - he will strip out script tags before sending it to the server and we'll also add some validation on the backend to ensure a higher level of security. 

There's some ideas here as well that we can investigate further. 
https://digi.ninja/blog/svg_xss.php

Reply:
To make sure that I understand, this PR once merged won't be available unless we flip a feature?  I definitely am uncomfortable pushing out a "trust the SVG until we have a mitigating PR."

My blocker on this is not on the implementation but to make sure that I understand the stakes and mitigation.

Reply:
> To make sure that I understand, this PR once merged won't be available unless we flip a feature?

Correct, we've been developing this feature in increments and. hiding the pages behind feature flags.